worldedit-core/src/main/java/com/fastasyncworldedit/core/command/brush/MovableTool.java

```diff
@@ -1,4 +1,4 @@
-package com.fastasyncworldedit.core.object.brush;
+package com.fastasyncworldedit.core.command.brush;
```

Comment:
is it a brush or a tool?

Reply:
It's a tooling interface like ScrollTool or ResetableTool.